src/mbed_tools/project/mbed_program.py

```diff
@@ -116,9 +116,9 @@ def resolve_libraries(self) -> None:
         """Resolve all external dependencies defined in .lib files."""
         self.lib_references.resolve()
 
-    def checkout_libraries(self, force: bool = False) -> None:
+    def deploy_libraries(self, force: bool = False) -> None:
```

Comment:
`deploy_libraries` doesn't make sense. We should just leave the internal APIs and only change the command name and handler.